### PR TITLE
Suggestion to improve api-first principles.

### PR DESCRIPTION
**Pull request by tkrop**
Stumbled over some formulations when writing our newsletter.

### CODE REVIEW

**Comment by maxim-tschumak** [comment]
:+1:

File: chapters/design-principles.adoc [hunk 1]
```diff
@@ -15,10 +15,9 @@ the design of our APIs should be based on the API as a Product
 principle:
 
 * Treat your API as product and understand the needs of its customers
-* Take ownership and advocate for the customer and continuous
-improvement
-* Emphasize easy understanding, discovery and usage of APIs; design APIs
-irresistible for client engineers
+* Take ownership and put yourself into the place of your customers
```
Comment (tfrauenstein) [inline chapters/design-principles.adoc]: thx. for the initiative; let's change 1st and 2nd item into...
* Treat your API as product and act like a product owner 
* Put yourself into the place of your customers; be an advocate for your customers needs

**Comment by tfrauenstein** [comment]
👍

**Comment by tkrop** [comment]
:+1:

**Pull request merged**